dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubSourceCodeRepo.java

```diff
@@ -88,6 +89,7 @@
 public class GitHubSourceCodeRepo extends SourceCodeRepoInterface {
 
     private static final Logger LOG = LoggerFactory.getLogger(GitHubSourceCodeRepo.class);
+    private static final String SHA_TYPE_FOR_SOURCEFILES = "sha1";
```

Comment:
Should maybe match string used in calcSHA1 and be `SHA-1`

Reply:
I put it as `sha1` because that's how they listed it in the link that's in this comment https://github.com/ga4gh/tool-registry-service-schemas/blob/develop/openapi/openapi.yaml#L601 but can still change it

Reply:
Good observation.
I'd still like it to match Java convention in our database and code, just have it convert when converting to TRS